src/shared/extensions.ts

```diff
@@ -1,9 +1,29 @@
 /*!
- * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * Copyright 2018-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Nit: 2018-2020

Reply:
this only satisfies the linter. We should change it to remove the year entirely. so we never have to change this again. Out of scope here though.

Reply:
? The linter shouldn't have a problem with a single year. If you're going through the trouble of editing this, we should at least make it current